content/en/infrastructure/containers/container_images.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Container Images View
+kind: documentation
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-container-image-view/"
+  tag: "Blog"
+  text: "Enhance your troubleshooting workflow with Container Images in Datadog Container Monitoring"
+- link: "/security/vulnerabilities"
+  tag: "Documentation"
+  text: "Cloud Security Management Vulnerabilities"
+- link: "/security/cloud_security_management/setup/csm_pro/?tab=aws#configure-the-agent-for-containers"
+  tag: "Documentation"
+  text: "Setting up container image vulnerabilities"
+- link: "/security/vulnerabilities/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting Cloud Security Management Vulnerabilities"
+---
+
+## Overview
+
+The [Container Images View][1] in Datadog provides key insights into every image used in your environment, helping you assess their deployment footprint, as well as detect and remediate security and performance issues that can affect multiple containers. You can view container image details alongside the rest of your container data, allowing you to troubleshoot image issues affecting infrastructure health. Additionally, you can view vulnerabilities found in your container images from [Cloud Security Management][2] (CSM) to help you streamline your security efforts.
+
+{{< img src="security/vulnerabilities/container_images.png" alt="The Container Images view highlighting vulnerabilities and container column sort feature" width="100%">}}
+
+## Configure Container Images View
+
+Images on the container images view are collected from several different sources (Live Containers, Image Collection, and Amazon ECR). The following instructions describe how to enable images from each of these sources.
```

Comment:
Is it Container Image view or container images view or Container Images View?
```suggestion
Images on the container images view are collected from several different sources (Live Containers, Image Collection, and Amazon ECR). The following instructions describe how to enable images from each of these sources.
```

Reply:
It is component of a product (Container Monitoring) -> so should be lower case, good call out though, let me go through the doc and make sure I have it consistent.